hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java

```diff
@@ -68,7 +68,7 @@ public EventType getEventType() {
      * @return the identifier of DistributedObject
      */
     public Object getObjectId() {
```

Comment:
shouldn't this method be renamed/deprecated as well? 
as now there is no obvious connection between `eventObjectId` and `distributedObjectName`.